functions/Get-DbaDbObjectTrigger.ps1

```diff
@@ -0,0 +1,97 @@
+function Get-DbaDbObjectTrigger {
+    <#
+    .SYNOPSIS
+        Get all existing database triggers on one or more SQL instances.
```

Comment:
Need to update references to `database triggers` to be `object-level triggers`

Reply:
Thanks Shawn! Will update tonight.